stash-scm-collector/src/main/java/com/capitalone/dashboard/support/DateUtils.java

```diff
@@ -0,0 +1,19 @@
+package com.capitalone.dashboard.support;
```

Comment:
Please move this to core module:
core/src/main/java/com/capitalone/dashboard/util